TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ftc16072/Mechanisms/Lift.java

```diff
@@ -14,6 +14,14 @@
 import java.util.List;
 
 public class Lift implements  Mechanism{
+    public enum LiftPositions{
+        FLOOR_POSITION,
+        LOW_POSITION,
+        MIDDLE_POSITION,
+        TOP_POSITION
+
+    }
+    public LiftPositions manipulatorPosition;
```

Comment:
Try to avoid public variables.   Instead a method that gets the manipulatorPosition is better.  That way someone can't accidentally change the value of it.